x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -198,3 +209,24 @@ export const getEndpointAuthzInitialState = (): EndpointAuthz => {
     canWriteEndpointExceptions: false,
   };
 };
+
+/**
+ * Duplicate logic to calculate if user has privilege to fetch Agent Policies,
+ * working only with Capabilities, in order to be able to use it e.g. in middleware.
+ *
+ * The logic works with Fleet granular privileges (`subfeaturePrivileges`) both enabled and disabled.
+ *
+ * @param capabilities Capabilities from coreStart.application
+ */
+export const canFetchPackageAndAgentPolicies = (capabilities: Capabilities): boolean => {
+  const canReadPolicyManagement = (capabilities.siem?.readPolicyManagement ?? false) as boolean;
```

Comment:
nit: `const canReadPolicyManagement = Boolean(capabilities.siem?.readPolicyManagement);`

Reply:
here I see why using `Boolean()` helps, as we can get rid off the nullish coalescing operator, but...

95eb39f43458117a6853594e5c081a4e9b9b78b3